xarray/tests/__init__.py

```diff
@@ -54,6 +54,7 @@ def _importorskip(modname, minversion=None):
 has_bottleneck, requires_bottleneck = _importorskip('bottleneck')
 has_rasterio, requires_rasterio = _importorskip('rasterio')
 has_pathlib, requires_pathlib = _importorskip('pathlib')
+has_seaborn, requires_seaborn = _importorskip('matplotlib')
```

Comment:
Scanning this quickly so forgive me if this isn't right - but should that be `_importorskip('seaborn')`?

Reply:
you are correct. (and my tests failed so I guess I'll revisit tomorrow)